uboot-mtk-20220606/arch/arm/dts/mt7986a-ruijie-x60-pro.dts

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+/dts-v1/;
+#include "mt7986.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+	model = "mt7986-ruijie-rg-x60-pro-v1";
+	compatible = "mediatek,mt7986", "mediatek,mt7986-rfb";
+
+	chosen {
+		stdout-path = &uart0;
+		tick-timer = &timer0;
+	};
+
+	config {
+		bootcmd = "mtkboardboot";
+		blink_led = "purple:status";
+		system_led = "white:status";
+	};
+
+	mtd-layout {
+		layout@0 {
+			label = "default";
+			cmdline = "console=ttyS0,115200n1 loglevel=8 firmware=0 uart_en=1";
+			mtdids = "nmbm0=nmbm0";
```

Comment:
does official use NMBM? based on the PR you submitted to upstream OpenWrt, the answer seems to be no.

Reply:
I forgot to change it.